week02/group_anagrams.cpp

```diff
@@ -0,0 +1,33 @@
+// TC: O(10^4*100) = O(n*n)
+// MC:O(10^4*100) = O(n*n) for using vector of type vector of string
+class Solution
+{
+public:
+    const static int MULTIPLIER = 31;
+    const static long long MOD = 32416190071;
```

Comment:
This is not a mersenne prime. So it is not recommended to use it here as it is not safe.